app-arch/tar/tar-1.34-r3.ebuild

```diff
@@ -37,6 +37,10 @@ PDEPEND="
 	app-alternatives/tar
 "
 
+PATCHES=(
```

Comment:
Please don't `git mv` for the ebuild, instead `cp` and then `ekeyword ~all` for the new ebuild.

Reply:
So here's the problem, the sdk I'm using (I'm a developer for "Container OS" which is a chrome os offshoot for servers) doesn't have all the tooling you'd expect on a typical gentoo system.

It's basically a barebones chroot of the gentoo userspace with bunch of stuff missing. Is there a way I could just manually set the keywords here? Do I just set them to `~all`?

Reply:
You just put a ~ prefix before every keyword in `KEYWORDS`.